packages/curve-ui-kit/src/widgets/Footer/Description.tsx

```diff
@@ -0,0 +1,47 @@
+import { t } from '@lingui/macro'
+import Box from '@mui/material/Box'
+import Link from '@mui/material/Link'
+import Typography from '@mui/material/Typography'
+
+import { RCLogoSM } from 'ui/src/images'
+
+export const Description = () => (
+  <Box
+    sx={{
+      display: 'flex',
+      flexDirection: 'column',
+      mr: '1.5rem',
```

Comment:
We can use the theme spacing.  `theme.spacing(6) == 1.5rem`
```suggestion
      marginRight: 6,
```